Live/Classes/Inventory.cpp

```diff
@@ -0,0 +1,9 @@
+#include "Inventory.h"
+
+bool Inventory::pickup( Item *item ) {
+    // TODO: Test for validity;
```

Comment:
I'm assuming you are gonna check here to see if there are too many items, and hence return false